src/python/platforms/fuchsia/device.py

```diff
@@ -214,6 +220,20 @@ def start_qemu():
 
 def stop_qemu():
   """Stop qemu."""
+
+  # Report whether qemu is running, for diagnostics
```

Comment:
Could we do this in fuzzers/libfuzzer.py instead, in `_restart_qemu`, as that's closer to the problem we're trying to diagnose?

Also, I just realised we already have a method using psutil to dump processes, so we can just do this instead:

```python
  def _restart_qemu(self):
    """Restart QEMU."""
    logs.log_warn(
        'Connection to fuzzing VM lost. Restarting.',
        processes=process_handler.get_runtime_snapshot())
```

Reply:
Ah, that's handy, thanks for pointing it out!